src/System.IO.FileSystem/tests/WinRT_BrokeredFunctions.cs

```diff
@@ -0,0 +1,207 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+using Windows.Storage;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    [SkipOnTargetFramework(~(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot))]
+    public partial class WinRT_BrokeredFunctions : FileSystemTest
+    {
+        private static string s_musicFolder = StorageLibrary.GetLibraryAsync(KnownLibraryId.Music).AsTask().Result.SaveFolder.Path;
+
+        [Fact]
+        public void CopyFile_ToBrokeredLocation()
+        {
+            string testFile = GetTestFilePath();
+            File.Create(testFile).Dispose();
+
+            string destination = Path.Combine(s_musicFolder, "CopyToBrokeredLocation_" + Path.GetRandomFileName());
+            try
+            {
+                Assert.False(File.Exists(destination), "destination shouldn't exist before copying");
+                File.Copy(testFile, destination);
+                Assert.True(File.Exists(testFile), "testFile should exist after copying");
+                Assert.True(File.Exists(destination), "destination should exist after copying");
+            }
+            finally
+            {
+                File.Delete(testFile);
```

Comment:
Nit: this first delete doesn't hurt, but it also shouldn't be necessary, as any GetTestFilePath()s will be cleaned up by the base class